### ISSUE TITLE
The Banner component icons need relevant screen reader only alternate text

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#193]
The banner component icons need to have screen reader only labelling that allows blind users to understand the state of the banner component without any visual indicator.

In the following image, the checkmark icon should have a generic, consistent label. Something like 'Success!' or 'Success:'
<img width="587" alt="Success banner component" src="https://user-images.githubusercontent.com/3613392/63023289-cd371880-be72-11e9-901f-9ea054ac4cae.png">

The `<Icon/>` component needs a prop that allows a developer to pass in a label that would handle this markup relatively painlessly. Note, this *does* require that icons have a unique `id`, which is always a bit of a hairy proposition. IDs could be generated randomly, or managed in such a way that developers are very unlikely to duplicate id values.
[Matchbox Icon](https://github.com/SparkPost/matchbox/blob/master/packages/matchbox-icons/src/IconBase/IconBase.js)

## Resources

* An example of an [accessibly labelled inline SVG](https://codepen.io/hmig/pen/adQoOJ) note that a unique `id` is required for the element to associate the `<svg>` with the `aria-labelledby` attribute on the `<title>`

**Comment by jonambas** [comment]
Resolved in #246
